guides/common/assembly_using-boot-disks-to-provision-hosts.adoc

```diff
@@ -25,7 +21,3 @@ include::modules/proc_generating-a-host-boot-disk.adoc[leveloffset=+1]
 endif::[]
 
 include::modules/proc_deploying-ssh-keys-during-provisioning.adoc[leveloffset=+1]
```

Comment:
FYI If you remove the context from the ID of this module, there's gonna be a problem because this module is currently included in both network boot and boot disk assemblies.